app/editor/menus/block.tsx

```diff
@@ -111,6 +112,18 @@ export default function blockMenuItems(dictionary: Dictionary): MenuItem[] {
       icon: TableIcon,
       attrs: { rowsCount: 3, colsCount: 3 },
     },
+    {
+      name: "math_inline",
+      title: dictionary.mathInline,
+      icon: MathIcon,
+      keywords: "math latex",
```

Comment:
```suggestion
      keywords: "math katex latex",
```

Reply:
Oops, forgot to commit this one, but I'll leave it for the next commit.